modules.json

```diff
@@ -109,21 +79,6 @@
       "filename": "htmlwidgets-1.7.2.omod"
     },
     {
-      "name": "ID Generation",
```

Comment:
I think we need this module for our identifier creation? Or how are identifier created?